identity/client/src/components/layout/AppHeader.tsx

```diff
@@ -1,10 +1,20 @@
 import { C3Navigation } from "@camunda/camunda-composite-components";
 import { useGlobalRoutes } from "src/components/global/useGlobalRoutes";
 import { useNavigate } from "react-router";
+import { useApi } from "src/utility/api";
+import { checkLicense } from "src/utility/api/headers";
 
 const AppHeader = () => {
   const routes = useGlobalRoutes();
   const navigate = useNavigate();
+  const { data: license } = useApi(checkLicense);
+  const showLicense: boolean =
+    license == null
+      ? false
+      : license.licenseType !== undefined &&
+        license.licenseType == "self-managed";
```

Comment:
`showLicense` should be looking for `saas`, that's the only case where it the license would be hidden. Otherwise, it would show

So, a small change is needed here

```
const showLicense: boolean =
    license == null
      ? false
      : license.licenseType !== undefined &&
        license.licenseType != "saas";
```